slashCommands/vote.ts

```diff
@@ -145,14 +163,26 @@ async function printVotes(pro: Set<string>, con: Set<string>, reply: Message, ti
         msgEmbed.setDescription(title);
     }
     //show embed
-    await reply.edit({embeds:[msgEmbed]})
+    await reply.edit({embeds:[msgEmbed], components:[]})
+}
+
+function playercanstartvote (interaction: CommandInteraction, messagerolepos: number): boolean {
```

Comment:
- I think we should use the term "user"/"member" over "player"
- I think this method should either get a user, or just the list of roles, since it doesn't need anythine else